### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by webseitz**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @webseitz. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/09/27 14:50:31 Loaded 6210 libraries from DB
Changing URL of library ScioSense_APC1 from https://github.com/sciosense/APC1_driver.git to https://github.com/sciosense/apc1-arduino.git
Success!
```
Done:
```
2023/09/27 14:52:12 Loaded 6210 libraries from DB
Changing URL of library ScioSense_ENS21x from https://github.com/sciosense/ENS21x_driver.git to https://github.com/sciosense/ens21x-arduino.git
Success!
```
Done:
```
2023/09/27 14:47:13 Loaded 6211 libraries from DB
Removing ScioSense ENS220
Success!
```

**Pull request merged**
